### ISSUE TITLE
Is there a circuit diagram?

### ISSUE THREAD

**Issue by sls-ca** [neonbjb/RaspPiArinc429#1]
This is a very interesting project.  Do you have circuit diagram from how you interfaced those chips with the PI.  Could you share that?

Thanks

**Comment by neonbjb** [comment]
Hi, 
Thanks! Unfortunately I dropped this project before I finished it. IIRC when I left it I was receiving data from the MCP23017. I'm more of a software engineer, so I did most of my electrical "designing" on paper and never took a picture or anything. I don't think it would be terribly hard to reverse engineer, though. The pin mapping from the DEI1016 to the PI is documented in code https://github.com/neonbjb/RaspPiArinc429/blob/master/src/com/applied/arinc/DEI1016Driver.java and the DEI1016 to the MCP23017 is fairly self explanatory - the data pins are just hooked up as you would expect from the datasheets.

**Comment by sls-ca** [comment]
Thanks for the reply... maybe if I get bored one weekend, I'll try to give this a shot.  Were the chips expensive?  I'm a software engineer myself, with minimal hardware experience.

**Comment by neonbjb** [comment]
I actually don't know. I contacted DEI about buying a chip and they sent me one for free instead for the cost of shipping.

My original plan was to make a cost effective NAV indicator for experimental planes. Thing is, the Garmin G5 and the Dynon equivalent came out right around the time I was working on this and they do everything I was planning on doing - but better, with more reliability then you'll get out of a Pi, and they aren't really all that expensive either.

**Comment by sls-ca** [comment]
yeah, I'm looking at some something more low level... maybe seeing if I could use it for a data concentrator bus, etc.  I've build something like this using more advanced hardware, which costs a lot of money, so this project was intriguing to me.  

Thanks again.
